src/main/java/net/mcreator/ui/modgui/codeviewer/ModElementCodeViewer.java

```diff
@@ -133,18 +133,18 @@ private synchronized void reload() {
 						}
 					}
 
-					for (File file : cache.keySet()) {
+					cache.keySet().stream().toList().forEach(file -> {

```

Comment:
Is there any intention for foreach here, or is this just for cleanup?

Reply:
This helps to avoid `ConcurrentModificationException`s (they are not update errors really).

Reply:
Using stream will not solve this, though, but nice catch nevertheless

Reply:
Actually it will due to toList :)